### ISSUE TITLE
(Feature request) Link to AWS API docs from cognitect.aws.client.api/doc ?

### ISSUE THREAD

**Issue by jffry** [cognitect-labs/aws-api#108]
# Description

AWS has usually-helpful API documentation outlining the purpose of API docs, and the meaning of (and caveats for) request/response values - for example, https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html

Is that link knowable/constructable from the underlying data that drives this library's description of available operations? If so, that could be a helpful addition to `cognitect.aws.client.api/doc` , a la:

```clojure
(require '[cognitect.aws.client.api :as aws])
(def s3 (aws/client {:api :s3}))
(aws/doc s3 :PutObject)

;=> -------------------------
;=> PutObject
;=> 
;=> Adds an object to a bucket.
;=> https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html
;=> 
;=> -------------------------
;=> Request
;=> ...
```

**Comment by dchelimsky** [comment]
I would love this feature myself!

Unfortunately the URLs are not provided in the data from which we build services (https://github.com/aws/aws-sdk-js/tree/master/apis), nor is there is consistency in the URLs across services, e.g.

```
https://docs.aws.amazon.com/kms/latest/APIReference/API_ListKeys.html
https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListUsers.html
https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBuckets.html
https://docs.aws.amazon.com/lambda/latest/dg/API_ListFunctions.html
```

The filenames appear consistent, but the paths do not.

**Comment by dchelimsky** [comment]
Actually, the service data for s3 does include `documentationUrl`s for its operations, but that's the only one. I've [updated the `api/doc-str`](96e510cc9f9b32789a56bb77b5f5e12910ccddd2) (used by `api/doc`) to present the `documentationUrl` when available.

I've also added [an issue to the aws-sdk-js project](https://github.com/aws/aws-sdk-js/issues/2977) requesting `documentationUrl` be added for every op on every service. As that is addressed, we'll see the docs appear in aws-api for other services.

**Comment by jffry** [comment]
Bummer, but hopefully upstream will add that everywhere. Thanks!

**Comment by dchelimsky** [comment]
Partial fix released in 0.8.408
